lib/statsd/instrument/environment.rb

```diff
@@ -4,8 +4,8 @@
 # which this library is active. It will use different default values based on the environment.
 class StatsD::Instrument::Environment
   class << self
-    def from_env
-      @from_env ||= StatsD::Instrument::Environment.new(ENV)
+    def current
```

Comment:
`current` is a bit weird of a name IMO. As it suggest it is likely to change like `User.current` would.

IMO something like `default` or `main` would make more sense.

Reply:
I named it `current` because it is the "current environment". I don't like `default`, because that implies it's something hardcoded, which is not the case. Not too sure about `main`